main.py

```diff
@@ -5,29 +5,24 @@
 from dotenv import load_dotenv
 from fastapi import FastAPI
 
-import helpers.commands
+import background_tasks.thread_worker
```

Comment:
so instead of a static thread_worker "class" I'd think something like this would work well:

```python
command_queue = CommandQueue(args)
threading.Thread(target=command_queue.start, ...).start()

/.../

...
DiscordCommands(..., job_queue=command_queue)
...

```